examples/change_baudrate.py

```diff
@@ -0,0 +1,100 @@
+from typing import Optional
+
+from ingenialink import CAN_BAUDRATE, CAN_DEVICE
+
+from ingeniamotion.motion_controller import MotionController
+
+
+def establish_canopen_communication(
+    mc: MotionController,
+    device: CAN_DEVICE,
+    channel: int,
+    baudrate: CAN_BAUDRATE,
+    dictionary_path: str,
+    node_id: Optional[int],
+) -> bool:
+    """Establish a CANopen communication.
+    
+    Find all available nodes by means of a scanning, and perform a communication.
+    If there isn't a selected node ID, it will perform a communication with the first found node ID.
+
+    Args:
+        mc: The object where there are all functions to establish a communication.
+        device: The type of transceiver (KVASER, PCAN, IXXAT).
+        channel: CANopen channel.
+        baudrate: The bit-timing rate of the communication.
+        dictionary_path: The absolute path where is placed a xdf file.
+        node_id: The selected node ID.
+
+    Returns:
+        True if the communication is performed successfully, False if it couldn't be established.
```

Comment:
That is confusing because if node_id is wrong will raise an exception instead if returns False. In my opinion, if it can't connect an exception should be raised.
In addition, I think that the node_id should be mandatory. It’s weird to change the baudrate to the first node_id found, and it’s highly discouraged.
Finally, with these changes, `establish_canopen_communication` function does not add value to the example, you can simply use `mc.communication.connect_servo_canopen`.